tests/resources/valid_taxpayer.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
please keep the namespace and other meta data elements like `header`, `funcCode`, `software`, etc